src/libraries/Microsoft.Extensions.Options/src/OptionsBuilderValidationExtensions.cs

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    /// <summary>
+    /// Extension methods for adding configuration related options services to the DI container via <see cref="OptionsBuilder{TOptions}"/>.
+    /// </summary>
+    public static class OptionsBuilderValidationExtensions
```

Comment:
The word `Validation` can be dropped for this class, (suggesting rename to `OptionsBuilderExtensions`).
This allows any future extensions methods (non-validation specific) to also make sense to be added here.

Just awaiting approval from .NET API reviewers before changing this on the PR.